### DISCUSSION TITLE
Game overs

### DISCUSSION THREAD

**Discussion by AcidicPancakes** [endless-sky/endless-sky#9874] [Balance, Mechanics, and Interactions]
Game Over

While playing endless sky I noticed two things lacking, one of them is music (i did see a pull request for it) and a game over screen.

Since I have no knowledge of programming, I don't know how hard it would be to have it that when your flagship gets destroyed or you make the wrong decision (builders) a prompt appears that would go like this

   									 

   				                                                        		      Game over
   							 
 				                                                              you have died would you like to
  					                                                       restart at your last checkpoint?
   				 
   			                                                	 restart at          	     exit to              	         exit
   		                                                 	    last checkpoint   	  main menu   		game

That should be all. I hope you all consider adding a game over screen since when I first died in Endless Sky it was a little weird for me to press the Esc key and not just use a menu.

**Comment by AcidicPancakes** [comment]
sorry i copied and pasted most of this from google docs for spell check reasons and it ended up looking like this here is the original

https://docs.google.com/document/d/1pUIs1HswtNX8UFKKZfFb0yAvKmfd8eqQq9ePEMVWc2U/edit?usp=sharing

**Comment by Hecter94** [comment]
Check out: #9717 and see if that's what you're looking for.

**Reply by AcidicPancakes** [reply]
yes this is it didn't come up when i was looking for any mention of what to do about a game over
